### PR TITLE
Add System.Net.IPAddress missing members

### LINKED ISSUE #12131: Port System.Net.IPAddress missing members
```csharp
    public class IPAddress {
        public long Address { get; set; }
    }
```

### PR DESCRIPTION
**Pull request by tijoytom**
Add back Address property , this's basically
resurrecting https://github.com/dotnet/corefx/pull/12595
since even after rebasing there's lot of merging to do.
It's easier to create a new PR then merging each file.

Fixes #12131

### CODE REVIEW

**Comment by tijoytom** [comment]
@davidsh Sorry i had to create a new PR since rebasing and merging  turned out to be night-mare . Can i get an LGTM on this  one?

**Comment by davidsh** [comment]
Test OuterLoop CentOS7.1 Debug Build and Test
Test OuterLoop OSX Debug Build and Test 
Test OuterLoop Ubuntu14.04 Debug Build and Test
Test OuterLoop Windows_NT Debug Build and Test

File: src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs [hunk 1]
```diff
@@ -279,5 +279,26 @@ public static void Equals_Compare_Success()
             Assert.True(ip4.GetHashCode().Equals(ip5.GetHashCode()));
             Assert.False(ip4.GetHashCode().Equals(ip6.GetHashCode()));
         }
+
+#if NetStandard17
```
Comment (stephentoub) [inline src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs]: cc: @weshaggard 
I believe we're aiming to put these tests into their own file that's conditionally compiled in.

**Comment by tijoytom** [comment]
@dotnet-bot  retest all

**Comment by tijoytom** [comment]
@dotnet-bot  test  Innerloop Linux ARM Emulator Debug Cross Build

**Comment by tijoytom** [comment]
Test OuterLoop Windows_NT Debug Build and Test

**Comment by tijoytom** [comment]
Test OuterLoop CentOS7.1 Debug Build and Test
Test OuterLoop OSX Debug Build and Test 
Test OuterLoop Ubuntu14.04 Debug Build and Test

**Comment by tijoytom** [comment]
Test Innerloop Ubuntu14.04 Release Build and Test

**Comment by tijoytom** [comment]
Test  Innerloop Windows_NT Release Build and Test

**Comment by tijoytom** [comment]
Test  Innerloop CentOS7.1 Debug Build and Test

**Comment by tijoytom** [comment]
Test Innerloop Windows_NT Debug Build and Test

File: src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs (hunk 1 above)
Comment (davidsh) [inline src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs]: > remember david mentioning that new test files should not be added , but conditional include the test in the existing file as i did

I made that comment before I was told that adding new files the "_netstandard1.7" suffix was the agreed-upon standard.  I was also told that it was a temporary thing and that soon these types of differences will be removed as the [master] branch moves toward only building for netstandard1.7.

**Comment by tijoytom** [comment]
Test Innerloop CentOS7.1 Release Build and Test

**Comment by tijoytom** [comment]
Test Innerloop OSX Debug Build and Test

**Comment by tijoytom** [comment]
Innerloop OSX Release Build and Test

**Comment by tijoytom** [comment]
Test  Innerloop Ubuntu14.04 Debug Build and Test

**Comment by tijoytom** [comment]
Test OuterLoop CentOS7.1 Debug

**Comment by tijoytom** [comment]
Test Innerloop OSX Release Build and Test

**Comment by tijoytom** [comment]
Test Innerloop Linux ARM Emulator Release Cross Build

**Pull request merged**